README.md

```diff
@@ -1 +1,118 @@
-# usaspending-api
\ No newline at end of file
+# USASpending API
+
+This is the API that will drive the new USAspending.gov. It is currently under active development and not stable. 
+
+## Getting Started
+
+This API is built on Python 3.3+, Django, and Django Rest Framework. 
+
+Assumptions:
+
+* You're able to install software on your local machine
+* You have git installed on your machine and are able to clone code repositories from GitHub. If this isn't the case, the easiest way to get started is to install [GitHub Desktop](https://desktop.github.com/ "GitHub desktop"), available for Windows or Mac.
+* You're familiar with opening a terminal on your machine and using the command line as needed.
+
+### Install PostgreSQL
+
+[PostgreSQL](https://en.wikipedia.org/wiki/PostgreSQL) is an object-relational database management system (ORDBMS) with an emphasis on extensibility and standards-compliance.
+
+1. Download the correct PostgreSQL installer for your operating system from [EnterpriseDB](http://www.enterprisedb.com/products-services-training/pgdownload) (we recommend PostgreSQL 9.4.x).
+2. Run the installer. As you proceed through the installation wizard, note your choices for port number, username, and password. You will need those when creating the broker's configuration file.
+
+More complete install documentation is available on the PostgreSQL [wiki](https://wiki.postgresql.org/wiki/Detailed_installation_guides).
+
+**Note:** If you're setting up the broker on Mac OSX, we recommend using [homebrew](http://brew.sh) to install PostgreSQL.
+
+### Install Python and Create Virtual Environment
+
+The broker's backend components currently run on Python 3.x. These instructions will walk you through the process of installing Python and creating a Python-based virtual environment to house the API. A virtual environment will isolate the broker software and its libraries from those running on your local system and prevent potential conflicts.
```

Comment:
"The broker" is mentioned a few times in this doc, which I don't _think_ you want?